### ISSUE TITLE
Starbases not showing up in Corporation view?

### ISSUE THREAD

**Issue by Raikia517** [eveseat/seat#116]
* **Problem**: "Starbases" is not found on the corporation summary page anymore, even with admin user.  Also, browsing to '/view/starbases/{id}' gives a 404
* **Expected**: A view of the starbase fuel info and silo info
* **Logs / Screenshots / Proof**: ![image](http://i.imgur.com/HBw9V1n.png)
* **Version Info**: Web version 1.0.2, just updated yesterday

**Comment by op6sie** [comment]
![image](https://cloud.githubusercontent.com/assets/15176561/16177134/fecb109c-3624-11e6-954a-8bdd13082a7f.png)

i have almost the same but with corp blueprints

**Comment by leonjza** [comment]
Latest web release is 1.0.27
https://github.com/eveseat/web/releases

**Comment by op6sie** [comment]
I have the latest version running.
before i reported the ommision.

Cheers.

**Comment by leonjza** [comment]
I can not reproduce this at all.

**Comment by op6sie** [comment]
register http://op6sie.myqnapcloud.com:4980
you can see there.
if you pm me an email i get you access to the software.

Cheers.

**Comment by op6sie** [comment]
or teamviewer is an option.

**Comment by leonjza** [comment]
Please only report software bugs here. For support you can join slack and ask there. I need specific steps to reproduce a bug.

**Comment by leonjza** [comment]
Also, @op6sie there is no blueprints tab.

**Comment by op6sie** [comment]
Ok then i make a new bug issue as assests is not showing all assets like
blueprints

Thank for the update.
Op 19 jun. 2016 6:06 p.m. schreef "Leon Jacobs" <notifications@github.com>:

**Comment by leonjza** [comment]
Blueprints seem to show fine too. Please stop posting to this ticket with unrelated bugs and try get support via the support channels first.

**Comment by Raikia517** [comment]
Right, so I provided the "Web version" number shown at the bottom of the page, which still shows "1.0.2".  I don't know if you are properly updating this number with each subsequent release.

Basically this bug has appeared after updating SEAT via these instructions: https://github.com/eveseat/seat/wiki/Upgrading-SeAT

For some reason, not all the top links (shown in the initial report) are available to users including the admin user who should be able to see all of them.  Was there some permissions change that would not be properly upgraded using those upgrade instructions?  I haven't done a fresh install of seat because I have over 450 APIs in the system right now with a lot of data.

I've briefly tried to track down the problem myself, but it is surprisingly more confusing since different parts of seat are broken up among different github repositories.  How is that top navigation bar populated with links?  I may be able to help bug hunt on my side

**Comment by leonjza** [comment]
So the version is definitely updated on releases [here](https://github.com/eveseat/web/blob/master/src/Config/web.config.php#L24). If you *really* are at version 1.0.2 at the moment, then it makes sense why you are not seeing the as they were added in version [1.0.12](http://seat-docs.readthedocs.io/en/latest/changelogs/web/#1012). That being said, the focus should then be why composer is not giving you the latest version of the `eveseat/web` dependency. You can find the local source file that has the version number somewhere like `/var/www/seat/vendor/eveseat/web/src/Config/web.config.php`.

As far as tracking the problem down, the history of the `composer.json` file [here](https://github.com/eveseat/web/commits/master/composer.json) shows how the version number requirements may have changed over time for the `web` package. The more importer `composer.json` though is the one in the base repository [here](https://github.com/eveseat/seat/blob/master/composer.json).

I *think* if you run `composer update -vv` with the `-v` args, it may be more descriptive as to why its not getting the latest version.

HTH

**Comment by Raikia517** [comment]
Well, I can confirm that that "web.config.php" file says version 1.0.2.  So I guess the problem is with the upgrade process:

>root@sunburst:/var/www/seat# php composer.phar update --no-dev -vv
Running composer as root/super user is highly discouraged as packages, plugins and scripts cannot always be trusted
> pre-update-cmd: php artisan clear-compiled
Loading composer repositories with package information
Updating dependencies
Dependency resolution completed in 1.488 seconds
Analyzed 8075 packages to resolve dependencies
Analyzed 129072 rules to resolve dependencies
Dependency resolution completed in 0.001 seconds
Generating autoload files
> post-update-cmd: php artisan optimize
Generating optimized class loader


Not much info in that, unfortunately.  But it does look like composer is actually installing 1.0.2....  I'm not incredibly familiar with composer, is there a way I can help you debug this?  Or something I can manually do to fix this?

**Comment by Raikia517** [comment]
I figured it out.  I changed composer.json to force version "1.0.12" (just to get ahead of 1.0.2), and it complained about not having php-gd installed.  So I installed php-gd and changed back to "1.*" in composer.json and it updated to 1.0.25 by itself.  So then to update to 1.0.27, I manually changed composer.json to say "1.0.27", and it complained about not having "php-curl" installed.  So I installed php-curl, and set composer.json back to "1.*", ran a composer update, and its now at 1.0.27.

So I guess if there is a dependency missing for a package, composer just silently pretends like there are no updates.  Thats pretty stupid...
